n_i2c.c

```diff
@@ -414,11 +399,9 @@ const char *i2cChunkedReceive(uint8_t *buffer, uint32_t *size, bool delay, uint3
         // Exit on timeout
         if (timeoutMs && (_GetMs() - startMs >= timeoutMs)) {
             *size = received;
-#ifdef ERRDBG
             if (received) {
-                NOTE_C_LOG_ERROR(ERRSTR("received only partial reply before timeout", c_iobad));
+                NOTE_C_LOG_DEBUG(ERRSTR("received only partial reply before timeout", c_iobad));

```

Comment:
Why the downgrade from `ERROR` to `DEBUG`?

IIRC, this means the response is invalid, and the transaction must be resubmitted.

I understand downgrading if it is in a retry loop, so long as it is eventually reported as an error once the retries have been exhausted.